src/flake8_comprehensions/__init__.py

```diff
@@ -199,6 +200,21 @@ def run(self) -> Generator[tuple[int, int, str, type[Any]], None, None]:
                         type(self),
                     )
 
+                elif (
+                    node.func.id == "sorted"
+                    and num_positional_args > 0
+                    and (
+                        isinstance(node.args[0], ast.GeneratorExp)
+                        and isinstance(node.args[0].elt, ast.Name)
+                    )
```

Comment:
`and (... and ...)` can be rewritten `and ... and ...`

Reply:
Oh, OK.
I thinked, in this issue, I should verify if isinstance of ast.Name only if it's a GeneratorExp 
Thanks

Reply:
the form `and ... and ...` already mean that. If the first condition is falsy, the second won't be checked.

Reply:
Oh, but yes... 🤦‍♀️